src/main/java/animals/service/JsonFileService.java

```diff
@@ -0,0 +1,18 @@
+package animals.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+
+public class
+JsonFileService extends FileService{
+    @Override
+    public ObjectMapper getObjectMapper() {
+        return new JsonMapper();
+    }
+
+    @Override
+    public String getFileName() {
+        String local = System.getProperty("user.language");
```

Comment:
This logic is duplicated in each FileService implementation. 
It is better to move to FileService. JsonFileService will return only extension "json"